src/gradata/detection/mode_classifier.py

```
@@ -0,0 +1,96 @@
+    "chat": set(),
+}
+
+# ── Public API ────────────────────────────────────────────────────────
+
+
+def classify_mode(prompt: str) -> tuple[str, float]:
+    """Classify user prompt into a mode.
+
+    Returns:
+        (mode, confidence) where mode is one of "code", "email", "config",
+        "documentation", "chat" and confidence is 0.0–1.0.
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)